lnrpc/rpc.proto

```diff
@@ -1475,9 +1475,10 @@ message NodeAddress {
 message RoutingPolicy {
     uint32 time_lock_delta = 1 [json_name = "time_lock_delta"];
     int64 min_htlc = 2 [json_name = "min_htlc"];
-    int64 fee_base_msat = 3 [json_name = "fee_base_msat"];
-    int64 fee_rate_milli_msat = 4 [json_name = "fee_rate_milli_msat"];
-    bool disabled = 5 [json_name = "disabled"];
+    uint64 max_htlc = 3 [json_name = "max_htlc"];
```

Comment:
To reduce the size of this PR, I suggest this change can be done in a follow-up! 😄 

(proto fields must be given new numbers btw).

Reply:
Fixed in #2434